README.md

```diff
@@ -21,239 +21,131 @@ under the License.
-
-## Automation Test
-Test cases utilizing 'dm' provided in the functional_test/ sub-directory. The test results will save a tarball file and locates in the "results" directory. They can execute those test cases through Makefile
-```shell
-#> cd demo_test/functional_test
-#> make test IP1=<ip of 1st device> PORT1=<RF port of 1st device> IP2=<ip of 2nd device> PORT2=<RF port of 2nd device>
-             USER1=<user of 1st device> PWD1=<password of 1st device> USER2=<user of 2nd device> PWD2=<password of 2nd device>
-```
-The test case could specific by the "TESTSDIR" option (for exmaple: tests/account_service)
-```shell
-#> cd demo_test/functional_test
-#> make test IP1=<ip of 1st device> PORT1=<RF port of 1st device> IP2=<ip of 2nd device> PORT2=<RF port of 2nd device>
-             USER1=<user of 1st device> PWD1=<password of 1st device> USER2=<user of 2nd device> PWD2=<password of 2nd device>
-             TESTSDIR=<test case directory>
-```
-## Manual testing at command line
-The 'dm' test tool needs to build at the command line the following by
-```shell
-#> cd demo_test/functional_test
-#> make dm
-```
-For running 'dm', please make and launch 'demotest' first.
-If you want to know the user manual, please read the 'demo_test/functional_test/README'
-
-# Reset k8s environment
-The command is removing all pods and helm chart.
-```shell
-#> make reset-pods
-```
+# Device Manager functionality
+Device Manager retrieves and collects information from devices (Redfish compliant) by using **Redfish** API.
+It uses ODIM's services to present data on northbound API.
+
+# Hardware requirements
+The Device Manager supports running the *Ubuntu Desktop 18.04 version*. For a single node deployment minimum requirement are
+- CPU 8 cores,
+- RAM 8GB,
+- Storage of at least 40GB.
+
+# Prerequisites
+Before building Device Manager, the following steps are required:
+- Go 1.17.10 installed,
+  - To install Go and to download necessary packages together with libraries for building Device Manager, run:
+  ```shell
+  $ make go-install
+  $ make prereq
+  $ source ~/.bashrc
+  ```
+- Docker 20.10.18 (minimum version) installed,
+- Certificates delivered or generated and placed into (project root)/build/certs (create this folder if necessary) (scripts for generation can be found [here](https://github.com/ODIM-Project/ODIM/tree/main/build/cert_generator)),
+- Config changes to fit your needs:
+  - **Device Manager**
+    - (project root)/svc-device-manager/config/config.yml
+  - **ODIM services**
+    - (project root)/lib-utilities/config/odimra_config.json
+  - **Redis**
+    - (project root)/build/redis/redis.conf
+  - **Etcd**
+    - (project root)/build/etcd/etcd.yml
+
+## Install Device Manager using single command
+Use this command to install and run Device Manager together with ODIM services, Redis and Etcd as Docker containers.
+
+```shell
+$ make all
+```
+
```

Comment:
Add paragraph - "Once that make is complete, the Redfish interface will be available at 127.0.0.1:45000. The port number is specified by the odimra_config.json file, property APIGatewayConf.Port.  The Redfish interface is used to create a connection to Device Manager."

Reply:
Added